docs/appendix.rst

```diff
@@ -524,8 +509,145 @@ To fix this, install the Python "wheel" package::
+
+      previously:
+
+      urllib3>=1.26.18,<2.0.0; python_version >= '3.7' (pywbem 1.6.x)
+
+The urllib3 Python package version 2.0 made a significant number of changes
+including the following list that can be potentially non-compatible:
+
+* Urllib3 >= 2.0 limits the minimum OpenSSL version >= 1.1.1.
+
+* Urllib3 >= 2.0 changes the default minimum TLS version to TLS 1.2 (previously was TLS 1.0)
+
+* Urllib3 >= 2.0 removed support for LibreSSL, WolfSSL and other SSL
+  implementations.  Multiple operating systems including: macOS and BSD use
+  LibreSSL as the TLS/SSL implementation. LibreSSL support however, was
+  restored to urllib3 in version 2.0.3.
+
+* Urllib3 >= 2.0 changed the minimum version for urllib3 version 2+ to Python 3.7.
+
+* Urllib3 >= 2.0 removed the default set of TLS ciphers, instead now urllib3
+  uses the list of ciphers configured by the system.
+
+Pywbem uses the requests dependent package which uses the urllib3 package which
+uses the pyopenssl to communicate with OpenSSL;
+neither pywbem nor requests pins the urllib3 package to versions below version 2.0.
+This is a function of the SSL implementation and pyopenssl.
+
+# TODO: This one bothers me.  It is the lib/ssl module which is a TLS/SSL wrapper
+# for socket objects that implemented PEP 644,  however, we never had any issues
+# with new minimum TLS version until 3.1.12 so the path through requests was
+# not impacted or we would have had to add urllib 2.0 in earlier tests.  It appears
+# that for the paths we use, it is requests, urllib3, pyOpenSSL packages and
+# OpenSSL version that are the issue.  I need to do another code review.
+Note that Python starting version 3.10 requires OpenSSL version 1.1.1 or higher
+(see Python PEP-644).
+
+Because of this change issue (Warnings, Exceptions, etc.) can occur in the
+process of:
+
+* pywbem package installation
+* pywbem package update ( ie.e updating urllib3)
+* OS update which changes the SSL implementaton version
+* Pywbem attempting to communicate with a WBEM server where the server may
+  not implement the new TLS version requirements.
+
+The following documents present more information on this change to urllib3 and
+the issues and possible corrections:
+
+* `Python PEP 644 – Require OpenSSL 1.1.1 or newer <https://peps.python.org/pep-0644/>`_
+  which defines the future direction of SSL in the Python world.
+
+* `Urllib3 migration guide <https://urllib3.readthedocs.io/en/stable/v2-migration-guide.html>`_
+
+* `macOS issues with Urllib3 - urllib3 issue 3024 <https://github.com/urllib3/urllib3/issues/3020>`_
+
+* `urllib3 issue 2168, Drop support for OpenSS\<1.1.1 <https://github.com/urllib3/urllib3/issues/2168>`_
+
+The current version and implementation  of the SSL library can be determined
+as follows::
+
+  $ python -c "import ssl; print(ssl.OPENSSL_VERSION)"
+  OpenSSL 1.1.1  7 Feb 2023
+
+  or
+
+  $ /usr/bin/python3 -c "import ssl; print(ssl.OPENSSL_VERSION)"
+  LibreSSL 2.8.3
```

Comment:
Can we stay consistent with how Python is invoked, i.e. use just "python"?

Reply:
DONE